Documentation/operations/upgrade.rst

```diff
@@ -333,6 +333,17 @@ Removed Metrics/Labels
 * ``cilium_operator_ipam_release_ops`` is removed. Please use ``cilium_operator_ipam_ip_release_ops`` instead.
 * The label of ``status`` in ``cilium_operator_ipam_interface_creation_ops`` is removed.
 
+Helm Options
+~~~~~~~~~~~~
+
+The way Linux capabilities are configured has been revamped in this release. 
+All capabilities of every container in the ``cilium-agent`` DaemonSet is
+configured from Helm's values, defaulting to the old behavior. If you have not
+been using ``securityContext.extraCapabilities`` you do not need to do anything.
+
+If you were leveraging ``securityContext.extraCapabilities``, you need to review
+``securityContext.capabilities.cilium_agent``.
```

Comment:
Nit: This should likely be an item list, so we can append to the list of other elements need to be added to the doc before 1.13 gets released.

Reply:
I've looked at previous notes regarding Helm and thought "this was the way" as it's the only item and Helm notes are not really common.

I'll change it right away though